ext/openssl/openssl.c

```diff
@@ -1553,6 +1553,7 @@ PHP_MINIT_FUNCTION(openssl)
 	php_stream_xport_register("tlsv1.0", php_openssl_ssl_socket_factory);
 	php_stream_xport_register("tlsv1.1", php_openssl_ssl_socket_factory);
 	php_stream_xport_register("tlsv1.2", php_openssl_ssl_socket_factory);
+	php_stream_xport_register("tlsv1.3", php_openssl_ssl_socket_factory);
```

Comment:
this should be added only if compiled with OpenSSL 1.1.1 (meaning it needs preprocessing check)